parameter/ConfigurableDomain.h

```diff
@@ -89,14 +112,31 @@ class CConfigurableDomain : public CBinarySerializableElement
                                                       bool& bIsLastApplied,
                                                       std::string& strError) const;
 
-    // Domain splitting
-    bool split(CConfigurableElement* pConfigurableElement, std::string& strError);
+    /** Split the domain in two.
+     * Remove an element of a domain and create a new domain which owns the element.
+     *
+     * @param[in] pConfigurableElement pointer to the element to remove
+     * @param[out] infos useful information we can provide to client
+     * @return true if succeed false otherwise
+     *
+     * FIXME: You may want to remove me
```

Comment:
right (although one could argue that the documentation could first be added in a separate patch and then updated when adding the 'infos' argument ;)).